public/js/app.js

```diff
@@ -61,7 +61,7 @@ angular.module('BuddyPairApp', ['BuddyPairApp.controllers', 'BuddyPairApp.servic
     $locationProvider.html5Mode(true);
 
     $authProvider.google({
-      clientId: '83414652329-fajrap1l590te27mjc8jec8q393okmfv.apps.googleusercontent.com'
+      clientId: '83414652329-0ji7glc5i4ep5lvlohue7ne3ktga7gg2.apps.googleusercontent.com',
```

Comment:
Please remove your `clientId` from the changeset or move the `clientId` parameter to `/auth/config.js`.